src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/resourcetakes/view/ResourceTakesView.kt

```diff
@@ -0,0 +1,70 @@
+package org.wycliffeassociates.otter.jvm.app.ui.resourcetakes.view
+
+import javafx.collections.ListChangeListener
+import org.wycliffeassociates.controls.ChromeableTabPane
+import org.wycliffeassociates.otter.common.data.model.ContentType
+import org.wycliffeassociates.otter.common.domain.content.Recordable
+import org.wycliffeassociates.otter.jvm.app.ui.resourcetakes.viewmodel.TakesViewModel
+import java.util.EnumMap
+import tornadofx.*
+import org.wycliffeassociates.otter.jvm.utils.getNotNull
+
+class ResourceTakesView : View() {
+    private val viewModel: TakesViewModel by inject()
+    private val tabPane = ChromeableTabPane()
+
+    class ContentTypeToTabMap(map: Map<ContentType, TakesTab?>): EnumMap<ContentType, TakesTab>(map)
```

Comment:
Does the value need to be nullable?

Reply:
Oops, not anymore. It was leftover from before I made the `getNotNull()` function, when nulls were propagating too far.